x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -119,6 +119,10 @@ export class CspPlugin
                 'To use this feature you must upgrade your subscription or start a trial'
               );
             }
+
+            if (!isSingleEnabledInput(packagePolicy.inputs)) {
+              throw new Error('Only one enabled input is allowed per policy');
+            }
```

Comment:
this ensures that our package won't be installed with multiple enabled inputs. technically, the UI won't enable users to select more than one but in case it fails, we'd still like to prevent the actual installation.